camel-kamelet/src/generated/resources/org/apache/camel/component/kamelet/kamelet.json

```diff
@@ -0,0 +1,38 @@
+{
+  "component": {
+    "kind": "component",
+    "name": "kamelet",
+    "title": "Kamelet",
+    "description": "The Apache Software Foundation provides support for the Apache community of open-source software projects.\n    The Apache projects are characterized by a collaborative, consensus based development process, an open and\n    pragmatic software license, and a desire to create high quality software that leads the way in its field.\n    We consider ourselves not simply a group of projects sharing a server, but rather a community of developers\n    and users.",
```

Comment:
The description here is wrong,

Reply:
where it comes from ?

Reply:
It takes the javadoc from the endpoint class, and fallback to pom.xml description

Reply:
ah ok, so here it takes the description from the apache parent pom :) I'll fix it